hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -679,6 +697,18 @@
       name: chip_clkmgr_external_clk_src
       desc: '''Verify the clkmgr switches to the correct clk src during certain LC states.
 
+            On POR lc asserts lc_clk_byp_req, and de-asserts it when lc_program
```

Comment:
just to clarify lc does not always assert this request, so we'll need to specifically trigger it for this behavior.

Reply:
I think we should put the chip in raw / test_locked / rma state to verify that in these states, the clkmgr uses external source. Plus we need additional CSR write in otp_ctrl. 

@msfschaffner, we should add assertion checks in rtl to prove this never turns on in prod lc states (if not already done so).

Reply:
I corrected the text. It seems these should be assertions in the LC RTL or its DV collateral. I created https://github.com/lowRISC/opentitan/issues/7663 for this.